twistr/events.go

```diff
@@ -597,6 +727,22 @@ func PlayMissileEnvy(s *State, player Aff) {
 	   immediately. If it contains an opponent’s Event, use the Operations value
 	   (no Event). The opponent must use this card for Operations during their next
 	   action round. */
+	s.TurnEvents[MissileEnvy] = player
+	maxOps := 0
+	for _, c := range s.Hands[player.Opp()].Cards {
+		if c.Ops > maxOps {
+			maxOps = c.Ops
+		}
+	}
+
+	selected := SelectCard(s, player.Opp(), ExceedsOps(maxOps-1))
```

Comment:
hm, we don't want to do this if there's only one card. We may have to suck it up and loop through the hand twice.